server/shared/types/terms/samplelst.ts

```diff
@@ -1,4 +1,5 @@
-import { Term, TermWrapper, BaseQ } from '../termdb.ts'
+import { BaseTerm, BaseQ } from './term.ts'
+import { TermWrapper } from './tw.js'
```

Comment:
I think this is supposed to be `import { TermWrapper } from './tw'`

Reply:
Thanks Colleen the extension if added should be ts, good you noticed this. In some files the imports have the extension and in some not so I follow the format adopted.